omero/sysadmins/cli/index.txt

```diff
@@ -13,6 +13,7 @@ need.
     config
     admin
     usergroup
+    data-ownership
```

Comment:
I'd suggest we use the same strategy for naming here: either hypenated or camelcase. cc: @hflynn